src/data/markdown/docs/02 javascript api/11 k6-ws.md

```diff
@@ -3,6 +3,15 @@ title: "k6/ws"
 excerpt: "k6 WebSocket API"
 ---
 
+<Blockquote mod='info'>
+
+#### An extension with much better and standard API exists
+
+[xk6-websockets](https://github.com/grafana/xk6-websockets) implements [the WebSockets API living standard](https://websockets.spec.whatwg.org/). While the implementation isn't full, it uses a global event loop instead of local one.
```

Comment:
@mstoykov we should include a similar mention https://k6.io/docs/using-k6/protocols/websockets

Reply:
fixed in [fb800dc](https://github.com/grafana/k6-docs/pull/644/commits/fb800dcb7cbd1f37682656a19213b9995913d105)